CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@
   - [Pull Request Guidelines](#pull-request-guidelines)
   - [Tips](#tips)
     - [Generating code from Markdown](#generating-code-from-markdown)
-    - [Generate code from Markdown and vice versa](generate-code-from-markdown-and-vice-versa)
+    - [Generate code from Markdown and vice versa](#generate-code-from-markdown-and-vice-versa)
```

Comment:
Nice evidence to have the link checker

Reply:
agree, would have been very difficult to spot otherwise